src/pages/EditStudyPage.tsx

```
@@ -1,5 +1,6 @@
 import StudyFormContent from "@/components/study/StudyFormContent";
 import Header from "@/components/ui/Header";
+import { useToast } from "@/components/ui/useToast";
 import { StudyFormProvider } from "@/hooks/useStudyForm";
 import { useToast } from "@/components/ui/useToast";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'useToast'. Consider to delete it or rename it.

'useToast' is defined here:


(lint/suspicious/noRedeclare)